packages/bun-polyfills/src/global/index.ts

```diff
@@ -1,9 +1,17 @@
-import { version } from '../modules/bun.js';
+import type { BunFile } from 'bun';
+import { version, readableStreamToFormData } from '../modules/bun.js';
 import './console.js';
 import './process.js';
+import './htmlrewriter.js';
 import os from 'node:os';
 
-//? NodeJS Blob doesn't implement Blob.json(), so we need to polyfill it.
+//? Yet another globalThis alias, because why not right?
+Object.defineProperty(globalThis, 'self', {
+    get() { return globalThis; }, set(_) { },
+    enumerable: true, configurable: true,
+});
```

Comment:
☝️ This is more a browser space thing. Adding `self` is not a polyfill for node so I'd keep this as a separate PR feature.

Reply:
Is there a ifdef kind of way to add only for browser polyfilling? What we have should probably be split out into node, browser, common.

Reply:
At least at the time of having written that code, Bun had `self` defined on runtime, which means server-side Bun code could use it which is why that's there.